StickyDots/StickyDotsView.swift

```diff
@@ -56,28 +56,31 @@ public class StickyDotsView: UIView {
     @IBInspectable public var circleWidth: CGFloat = 10
     
 
-    public convenience init(frame: CGRect, attachedTo: UIScrollView? = nil) {
+    public convenience init(frame: CGRect, attachedTo: UIScrollView? = nil, indicatorColor:UIColor = .white, dotsWidth:CGFloat = 10, dotsColor:UIColor = .black, dotsAlpha:CGFloat = 0.5) {
         self.init(frame: frame)
         self.attachedView = attachedTo
+        self.circleWidth = dotsWidth
         
         let indicatorsWidth = CGFloat(2 * _numOfPages - 1) * circleWidth
         indicatorView.frame.size.width = circleWidth
+        indicatorView.frame.size.height = circleWidth
         indicatorView.center.x = circleWidth
+        indicatorView.center.y = circleWidth
         indicatorView.layer.cornerRadius = circleWidth * 0.5
-        indicatorView.backgroundColor = .white
+        indicatorView.backgroundColor = indicatorColor
         
         self.frame.size = CGSize(width: indicatorsWidth, height: circleWidth)
         
         let dotsView = UIView(frame: CGRect(x: 0, y: 0, width: indicatorsWidth, height: circleWidth))
         for i in 0...(_numOfPages - 1) {
             let dotView = UIView(frame: CGRect(
                 x: (circleWidth * 0.5) + (2 * circleWidth * CGFloat(i)),
-                y: circleWidth,
+                y: circleWidth * 0.5,
```

Comment:
What’s the purpose of adding this here?

Reply:
Changing the circle width misaligned the dots from the indicator. The positioning only worked with size 10. Adding a y position to the indicator equal to circle width, and changing the y position here fixed that.